modules/ding_event/ding_event.pages_default.inc

```diff
@@ -341,6 +343,8 @@ taxonomy/term/%term:tid',
   $display->cache = array();
   $display->title = 'Events';
   $display->uuid = 'e6d7ce53-caaa-4af3-8588-00d36e8d512a';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.